index.html

```diff
@@ -0,0 +1,68 @@
+  <script src="https://kit.fontawesome.com/ebaeaa9ace.js"></script>
+  <link rel="stylesheet" href="normalize.css">
+  <link rel="stylesheet" href="style.css">
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>Apple Clone</title>
+</head>
+<body>
+  <div class="container"> <!-- Container Start-->
+    <div class="container-top"> <!--Top Container Start-->
+      <header>
+          <div class="grid-container"> <!--Grid Container Start-->
+              <div class="grid-item"><i class="fab fa-apple"></i></div>
+              <div class="grid-item">Store</div>
+              <div class="grid-item">Mac</div>
+              <div class="grid-item">ipod</div>
+              <div class="grid-item">iPhone</div>
+              <div class="grid-item">iPad</div>
+              <div class="grid-item">iTunes</div>
+              <div class="grid-item">Support</div>
+              <div class="grid-item-search">
+                <form action=""> <!--Form Start-->
+                  <i class="fas fa-search"></i>
+                  <input type="text" value="     ">
+                </form><!--Form End-->
+              </div>
+          </div> <!-- Grid Container End-->
+        </header>
+    </div> <!--Top Container End-->
+    <div class="text-container"> <!-- Text container start-->
+      <div class="main-text-header">
+      </div>
+      <div class="secondary-text"> 
+      </div>
+      <div class="your-verse">
+      </div>
+      <div class="your-verse-border">
+      </div>
+    </div>  <!-- Text Container End-->
+  </div><!-- Container End-->
+  <div class="grid-other-topics"> <!-- Grid options start-->
+      <div class="topic-grid image_1"></div>
+      <div class="topic-grid image_2"></div>
+      <div class="topic-grid image_3"></div>
+      <div class="topic-grid image_4"></div>
+    </div> <!-- Grid Options End-->
+  <footer> <!--Main Footer Start-->
+    <div class="footer-left-container"><!--Footer left container start-->
+      <span class="footer-text">
+          Shop the <span class="footer-link">Apple Online Store</span> (1-800-MY-APPLE), visit an <span class="footer-link">Apple Retail Store</span>, or find a <span class="footer-link">reseller</span>.
+      </span>
+      <br>
+      <span class="footer-text">
+          Copyright © 2014 Apple Inc. All rights reserved. <span class="footer-link">Terms of Use | </span> <span class="footer-link">Privacy Policy</span>
+      </span>
+    </div><!--Footer left container end-->
+    <div class="footer-right-container"> <!-- Footer rigth container start-->
+      <span class="footer-text footer-link">Site | MapHot | NewsRSS | FeedsMedia | InfoEnvironment | Job Opportunities | Contact Us | <img src="images/img_usa.png" alt="" class="usa-logo"></span>
+    </div> <!-- Footer right container end-->
+  </footer><!--Main Footer End-->
+  <p>Test</p>
```

Comment:
You forgot to remove this "test" comment :D

Reply:
All Changes done XD